docs/home/howto/MODULES.md

```diff
@@ -2,99 +2,76 @@
 
 ## Introduction
 
-In version 0.8 of epiphany we are introducing modules. Modularisation of epiphany environment will result in: 
- * smaller code bases for separate areas, 
- * simpler and faster test process, 
- * interchangeability of elements providing similar functionality (eg.: different kubernetes providers), 
- * faster and more focused release cycle. 
- 
-Those and multiple other factors (eg.: readability, reliability) influence this direction of changes. 
+In version 0.8 of Epiphany we are introducing modules. Modularization of Epiphany environment will result in:
```

Comment:
Maybe:
In version 0.8 of Epiphany we introduce modules

Reply:
Or even 'introduced' :)